exampleCourse/questions/demo/computeNumber-fromArray/info.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
More mixed case in the QID. I'm fine with keeping it as is, if it's intentional.

Reply:
I did not do a detailed look at all the QIDs to check for consistency. I would prefer to not change QIDs at this moment, because it may affect documentation and other places.

Reply:
after talking with @nwalters512, we feel it will be better to keep the QIDs as is for now, and we will update all of them later for consistency at a later time.